deploy/kubernetes/examples/demo-snapshotclass.yaml

```diff
@@ -8,3 +8,4 @@ parameters:
   csi.storage.k8s.io/snapshotter-secret-name: demo-secret
   csi.storage.k8s.io/snapshotter-secret-namespace: default
   snapshot_name_prefix: demoSnapshot   # Optional.
+  pool: demo-pool # mandatory only in ds8k
```

Comment:
I'm not sure "ds8k" is an official term.
please use spacing similar to other yaml comments, start with a capital letter, etc...

Reply:
done